citesphere/src/main/java/edu/asu/diging/citesphere/web/user/DeleteLocalCopy.java

```diff
@@ -0,0 +1,38 @@
+package edu.asu.diging.citesphere.web.user;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.citesphere.api.v1.model.impl.Group;
+import edu.asu.diging.citesphere.core.exceptions.GroupDoesNotExistException;
+import edu.asu.diging.citesphere.core.service.IGroupManager;
+import edu.asu.diging.citesphere.core.user.IUserManager;
+import edu.asu.diging.citesphere.user.IUser;
+
+@Controller
+public class DeleteLocalCopy{
+     
+    @Autowired
+    private IGroupManager groupManager;
+
+    @Autowired
+    private IUserManager userManager;
+    
+    @RequestMapping(value = "/auth/group/{zoteroGroupId}/mongo", method = RequestMethod.DELETE )
```

Comment:
let's not call this `.../mongo`. That's implementation specific and won't necessarily always be true. Maybe `../{zoteroGroupId}/resync` or something. Also, let's make that a POST request as DELETE typically indicates deleting the resource at the given endpoint (at least in RESTful APIs), which is not the case here.